news/2024/2024-10-12-osu-originals-badly-wood-cup-2024.md

```diff
@@ -50,7 +50,7 @@ See what's in store for yourself:
     <source src="https://assets.ppy.sh/artists/441/Songs/seatrus%20-%20Blank%20society.mp3">
 </audio>
 
-### [katagiri - VIVID ABYSS](https://osu.ppy.sh/beatmapsets/2227434) mapped by [Altai](https://osu.ppy.sh/users/5745865)
+### [katagiri - VIVID ABYSS](https://osu.ppy.sh/beatmapsets/2227434) mapped by [Altai](https://osu.ppy.sh/users/5745865), [Lince Cosmico](https://osu.ppy.sh/users/6070370), [jas](https://osu.ppy.sh/users/8920907) and [toybot](https://osu.ppy.sh/users/2848604)
```

Comment:
```suggestion
### [katagiri - VIVID ABYSS](https://osu.ppy.sh/beatmapsets/2227434) mapped by [Altai](https://osu.ppy.sh/users/5745865), [Lince Cosmico](https://osu.ppy.sh/users/6070370), [jas](https://osu.ppy.sh/users/8920907), and [toybot](https://osu.ppy.sh/users/2848604)
```
:grimacing:

Reply:
i do believe we don't enforce the oxford comma in news posts like we do on the wiki